apps/root/src/services/sdkService.ts

```diff
@@ -472,7 +472,7 @@ export default class SdkService {
     const mockedEarnPositions = new Promise<SdkEarnPosition>((resolve) => {
       setTimeout(() => {
         resolve({ ...sdkDetailedEarnPositionMock, id: positionStrategyId });
-      }, 1000);
+      }, 0);
```

Comment:
Why? :( We can keep loading states

Reply:
Oh shit yeah, was debugging and forgot to unstage this changes